lnrpc/rpc.proto

```diff
@@ -537,6 +537,38 @@ message Utxo {
     int64 confirmations = 6;
 }
 
+enum OutputScriptType {
+    PUBKEY_HASH = 0;
+    SCRIPT_HASH = 1;
+    WITNESS_V0_PUBKEY_HASH = 2;
+    WITNESS_V0_SCRIPT_HASH = 3;
+    PUBKEY = 4;
```

Comment:
We're planning on upgrading the `grpc` library version soon. There have been some recent changes which will warn if enum value names inside the same gRPC package collide. So if we'd add another enum with a different name but the same `PUBKEY` value name for example, we'd get a nasty error. That's why I think we should start adding a prefix to all our (new) enums.
So something like this:

```
enum OutputScriptType {
    SCRIPT_TYPE_PUBKEY_HASH = 0;
    SCRIPT_TYPE_SCRIPT_HASH = 1;
    SCRIPT_TYPE_WITNESS_V0_PUBKEY_HASH = 2;
    SCRIPT_TYPE_WITNESS_V0_SCRIPT_HASH = 3;
    SCRIPT_TYPE_PUBKEY = 4;
...
```

I know that makes it impossibly long in the generated golang code. But migrating enum names because of a collision is even more tedious.

Reply:
Makes sense! I have changed it accordingly.